### PR TITLE
Refactor plugin - remove pikpay/webteh.hr

### LINKED ISSUE #16: Refactor plugin - remove pikpay/webteh.hr

### PR DESCRIPTION
**Pull request by 4050350194**
Refactored all matched instances of `pikpay` into `monri`.

closes #16

### CODE REVIEW

File: class-pikpay.php [hunk 1]
```diff
@@ -567,13 +573,13 @@ public function generate_form($order_id)
...
 
-        //Convert currency to match PikPay's requested currency
+        //Convert currency to match Monri's requested currency
         if ($currency == "KM") {
             $currency = "BAM";
         }
 
         //Generate digest key
-        $digest = hash('sha512', $this->pikpaykey . $order->get_id() . $order_total . $currency);
+        $digest = hash('sha512', $this->monri_key . $order->get_id() . $order_total . $currency);
```
Comment (jasminsuljic) [inline class-pikpay.php:582]: @4050350194 while you are at it - rename `monri_key` to `monri_merchant_key`.

File: class-pikpay.php [hunk 2]
```diff
@@ -596,21 +602,21 @@ public function generate_form($order_id)
 
             'language' => $this->form_language,
             'transaction_type' => $transaction_type,
-            'authenticity_token' => $this->pikpayauthtoken,
+            'authenticity_token' => $this->monri_auth_token,
```
Comment (jasminsuljic) [inline class-pikpay.php:605]: @4050350194 rename to `monri_authenticity_token`. Renaming keys will greatly reduce potential miss-understandings.

**Review by jasminsuljic — CHANGES_REQUESTED** [review]

**Comment by jasminsuljic** [comment]
@4050350194 do you have test environment for WooCommerce? I can send an official setup request to your's PM

File: class-pikpay.php (hunk 2 above)
Comment (4050350194) [reply class-pikpay.php]: Resolved by commit ID: 2ecebc1d36ff40711be89c5071d93220744b53a9.

File: class-pikpay.php (hunk 1 above)
Comment (4050350194) [reply class-pikpay.php]: Resolved by commit ID: 2ecebc1d36ff40711be89c5071d93220744b53a9

**Comment by 4050350194** [comment]
> @4050350194 do you have test environment for WooCommerce? I can send an official setup request to your's PM

@jasminsuljic I have provided test environment information on our conversation via email. CI/CD setup is still ongoing. We will notify you when this is ready.

You can provide us any additional specific setup information here or via email.

**Review by jasminsuljic — APPROVED** [review]

**Pull request merged**